lib/components/Table/Table.tsx

```diff
@@ -233,13 +234,21 @@ export const TableFooter: FC<TableFooterProps> = ({
     return "";
   }, [pageSize, pageSizeOptions, onPageSizeSelected]);
 
+  const firstTDColSpan = useMemo(() => {
+    if (!numberOfColumns) return 4;
+    if (numberOfColumns <= 4) return numberOfColumns - 1;
+    return 4;
+  }, []);
+
   return (
     <TFoot>
       <tr>
-        <td colSpan={4}>
-          {itemsPerPageSelector} {itemsCount}
-        </td>
-        <td colSpan={100}>
+        {(itemsPerPageSelector || itemsCount) && (
```

Comment:
this was causing problems if the table had less than 4 columns overall, making it to be out of alignment.
also, making it conditional helps in case we want to hide it and let it not interfere with the other TD